modules/panel.py

```diff
@@ -92,6 +92,8 @@ def __init__(self, window: sublime.Window):
 		settings.set('line_spacing', 0)
 		settings.set('context_menu', 'Widget Debug.sublime-menu')
 		settings.set('draw_centered', True)
+		settings.set('draw_unicode_white_space', 'none')
+		settings.set('draw_unicode_bidi', False)
```

Comment:
Disabling `draw_unicode_bidi` is what was needed to fix the layout but it also makes sense to disable `draw_unicode_white_space` IMO.